en/mkdocs.yml

```diff
@@ -87,6 +89,7 @@ nav:
     - 'Data Publishing':
        - 'Introduction': learn/data-publishing.md
        - 'Tryout': learn/data-publishing-tryout.md
+    - 'CDS Data Reporting': learn/cds-data-reporting.md
```

Comment:
Shall we use CDR Data Reporting - https://www.accc.gov.au/focus-areas/consumer-data-right-cdr-0/reporting-forms-rule-94?